wgengine/ifstatus.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// +build !windows
+
+package wgengine
+
+import (
+	"time"
+
+	"github.com/tailscale/wireguard-go/tun"
+	"tailscale.com/types/logger"
+)
+
+// Dummy implementation that does nothing.
+func waitIfaceUp(iface tun.Device, timeout time.Duration, logf logger.Logf) error {
```

Comment:
waitInterfaceUp

(iface is fine for the parameter name and internal type, but let's keep the function name explicit)